ui/components/app/account-menu/account-menu.component.js

```diff
@@ -31,7 +33,7 @@ import {
   ///: END:ONLY_INCLUDE_IN
 } from '../../../helpers/constants/routes';
 import TextField from '../../ui/text-field';
-import IconCheck from '../../ui/icon/icon-check';
+import IconSpeechBubbles from '../../ui/icon/icon-speech-bubbles';
```

Comment:
Should this be here? Doesn't look like it is used?

Reply:
Removed

Reply:
Do you need to push? I still see IconSpeechBubbles here